egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -1016,19 +1016,19 @@ def _test_scaled_lstm_forward():
     assert torch.allclose(c, c_chunk)
 
 
-def _test_grad_cliper():
-    grad_cliper = GradientCliper(
+def _test_grad_clipper():
+    grad_clipper = GradientClipper(
         grad_norm_threshold=10.0, grad_scale_factor=0.9, grad_max_norm=1.0
```

Comment:
I actually think the threshold thing may not end up being helpful... I don't know if I ever tested it independently from the max_norm. I think it will be very rare that it goes above the grad_norm_threshold.